Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/service/IConceptEditService.java

```diff
@@ -0,0 +1,23 @@
+package edu.asu.conceptpower.app.service;
+
+import edu.asu.conceptpower.app.bean.ConceptEditBean;
+import edu.asu.conceptpower.app.exceptions.IndexerRunningException;
+import edu.asu.conceptpower.app.exceptions.LuceneException;
+import edu.asu.conceptpower.core.ConceptEntry;
+
+public interface IConceptEditService {
+
+    /**
+     * This method edits the concept.
+     * 
+     * @param conceptEntry
+     * @param conceptEditBean
+     * @param userName
+     * @throws IllegalAccessException
+     * @throws LuceneException
+     * @throws IndexerRunningException
+     */
+    public void editConcepts(ConceptEntry conceptEntry, ConceptEditBean conceptEditBean, String userName)
```

Comment:
bad javadoc, you only say what the method name already tells me. what does this method do?